build.gradle

```diff
@@ -1,235 +1,1512 @@
+//version: 1685785062
+/*
+ DO NOT CHANGE THIS FILE!
+ Also, you may replace this file at any time if there is an update available.
+ Please check https://github.com/GTNewHorizons/ExampleMod1.7.10/blob/master/build.gradle for updates.
+ */
```

Comment:
Is this generated/boilerplate?

Reply:
Correct, this build.gradle file should never be touched by a human, in fact running Gradle commands will tell you when it needs to be updated and tell you how to do so, which is also just one command.  Don't edit this file because it will be overwritten on updates, that's what the other Gradle files are for.